package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "facebook-insight-stream",
-  "version": "3.0.1",
+  "version": "3.2.0",
```

Comment:
should be 4.0.0 because we switched a major api version

Reply:
Changed.